src/hist/basehist.py

```diff
@@ -155,54 +155,29 @@ def project(self, *args: Union[int, str]):
         """
         Projection of axis idx.
         """
-        if len(args) == 0 or all(isinstance(x, int) for x in args):
-            return super().project(*args)
-
-        elif all(isinstance(x, (int, str)) for x in args):
-            indices: tuple = tuple()
-            for name in args:
-                key = self._name_to_index(name) if isinstance(name, str) else name
-                if key in indices:
-                    raise ValueError("Duplicate index keys are contained")
-                else:
-                    indices += (key,)
-
-            return super().project(*indices)
-
-        else:
-            raise TypeError(
-                f"Only projection by indices and names is supported for {self.__class__.__name__}"
-            )
+        int_args = [self._name_to_index(a) if isinstance(a, str) else a for a in args]
```

Comment:
Just rebuild this, replacing string names with ints. The underlying `.project()` will handle repeated indices the same way manually repeated indices are handled (I don't really even know what `project(0, 0)` does, the C++ handles this)

Reply:
https://github.com/scikit-hep/hist/blob/6fb66d385bbc57c5dea7e2f9607df416242f9ad1/tests/test_general.py#L330-L331